app/src/androidTest/java/com/github/h3lp3rs/h3lp/ChatUiTest.kt

```diff
@@ -0,0 +1,81 @@
+package com.github.h3lp3rs.h3lp
+
+import android.app.Activity
+import android.content.Intent
+import android.app.Instrumentation.*
+import android.os.Bundle
+import androidx.test.core.app.ActivityScenario
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.contrib.RecyclerViewActions
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.Intents.init
+import androidx.test.espresso.intent.Intents.intending
+import androidx.test.espresso.intent.matcher.IntentMatchers.anyIntent
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import com.github.h3lp3rs.h3lp.messaging.*
+import com.xwray.groupie.ViewHolder
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+private const val CONVERSATION_ID = "testing_id"
+private const val SENT_MESSAGE = "Testing Chat UI"
+private const val RECEIVED_MESSAGE = "Tests succeeded!"
+
+class ChatUiTest {
+    private val currentMessenger = Messenger.HELPEE
+    private val toMessenger = Messenger.HELPER
+
+    private lateinit var conversationFrom: Conversation
+    private lateinit var conversationTo: Conversation
+
+    @get:Rule
+    val testRule = ActivityScenarioRule(ChatActivity::class.java)
+
+    @Before
+    fun setup() {
```

Comment:
I think you forgot to initiate the intent here

Reply:
It isn't needed as I am initializing the intent using the test rule